### PR TITLE
*fix missing column empty string

### PR DESCRIPTION
**Pull request by LeonadoRivaldo**
there is a issue on ur board tha tis about this, I add a else and changed the null check for a falsy check so now undefined or empty strings should be covered.

### CODE REVIEW

File: Angular-csv.ts [hunk 1]
```diff
@@ -173,15 +173,22 @@ export class AngularCsv {
...
 
         if (this._options.nullToEmptyString) {
-            if(data === null) {
+            if(!data) {
                 return data = '';
+            }else{
+                return data;
             }
-            return data;
         }
```
Comment (LeonadoRivaldo) [inline Angular-csv.ts:181]: this is real change

**Comment by LeonadoRivaldo** [comment]
@alhazmy13 hey man, sorry all those commits, the code will work the only real change is at  commit d84e143cd918f890f3a149bea9900c20f773f0f0

**Comment by LeonadoRivaldo** [comment]
it is just my VS code formated the code and i dindt saw so :P

File: Angular-csv.ts (hunk 1 above)
Comment (alhazmy13) [reply Angular-csv.ts:181]: LGTM

**Review by alhazmy13 — APPROVED** [review]
LGTM

**Comment by LeonadoRivaldo** [comment]
I need you to merge this coz i need to finish my project today

**Comment by alhazmy13** [comment]
Version [1.0.4](https://github.com/alhazmy13/angular-csv-ext/releases/tag/1.0.4) is live

**Pull request merged**